03_letters/n0npax/main_test.go

```diff
@@ -103,7 +103,7 @@ func helperTestSortLetters(name string, input map[rune]int, expected []string) f
 				t.Errorf("Test: %s - Expect: %s got: %s", name, expected[i], v)
 			}
 		}
-		assert.Equalf(t, len(expected), len(result), "Lenght missmatch")
+		assert.Equalf(t, len(expected), len(result), "Length missmatch")
```

Comment:
@juliaogris this line was changed due to: https://travis-ci.com/anz-bank/go-course/builds/116097682
seems new golanci-lint checking misspellings in strings :open_mouth:
If you prefer I can split this into 2 PR

Reply:
Did it catch "missmatch"?

Reply:
Yes. Link refers to previous build without modification of this particular file.

Reply:
I think what @anzdaddy is saying is that the word "missmatch" is also misspelt (unless it is a reference to the Swedish electronica band: https://en.wikipedia.org/wiki/MissMatch). It should be "mismatch". Even though golangci-lint is not picking this up, since you are fixing that line anyway, you may as well fix that other misspelling.

Reply:
awww I see now. Will be fixed soon